protos/bond_payload.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message BondPayload{
+    enum Action {
+        CREATE_ORGANIZATION = 0;
+        UPDATE_ORGANIZATION = 1;
+        UPDATE_ORGANIZATION_AUTHORIZATION = 2;
```

Comment:
It seems the DeleteOrganization action is missing?

Reply:
After taking out the organization_id in the DeleteOrganization message, I also took out that empty message and the associated action. Darian and I discussed the ability to have an action in the payload file without an actual message (which, if we did implement the delete org transaction we would probably need the org_id field in that message) so we thought it wasn't really necessary to implement that transaction.

Reply:
For me it makes sense to have the action without the message (if it is possible). I thought the reason we deleted the org_id in the DeleteOrganization message, was because we could determine the organization being deleted through the participant who submitted the transaction.